prow/config.yaml

```diff
@@ -24,10 +24,10 @@ plank:
         censoring_concurrency: 2 # 2 concurrent censoring
         censoring_buffer_size: 3092 # 3kB
       utility_images:
-        clonerefs: gcr.io/k8s-prow/clonerefs:v20220721-cf42b99a12
-        initupload: gcr.io/k8s-prow/initupload:v20220721-cf42b99a12
-        entrypoint: gcr.io/k8s-prow/entrypoint:v20220721-cf42b99a12
-        sidecar: gcr.io/k8s-prow/sidecar:v20220721-cf42b99a12
+        clonerefs: gcr.io/k8s-prow/clonerefs:v20220822-46c56dcda7
+        initupload: gcr.io/k8s-prow/initupload:v20220822-46c56dcda7
+        entrypoint: gcr.io/k8s-prow/entrypoint:v20220822-46c56dcda7
+        sidecar: gcr.io/k8s-prow/sidecar:v20220822-46c56dcda7
```

Comment:
Why is there this change?

Reply:
I was wondering myself, it was autogenerated by running:
go run development/tools/cmd/rendertemplates/main.go --data templates/data

Reply:
Should I revert this?

Reply:
OK I reverted those